module/data/item/templates/activated-effect.mjs

```diff
@@ -152,9 +156,9 @@ export default class ActivatedEffectTemplate extends SystemDataModel {
       this.parent.labels.range = range.filterJoin(" ");
     } else this.parent.labels.range = game.i18n.localize("DND5E.None");
 
-    if ( this.recharge ) this.parent.labels.recharge = `${game.i18n.localize("DND5E.Recharge")} [${
-      `${this.recharge.value}${parseInt(this.recharge.value) < 6 ? "+" : ""}`
-    }]`;
+    if ( this.recharge ) this.parent.labels.recharge = `${game.i18n.localize("DND5E.Recharge")} [
+      ${this.recharge.value}${parseInt(this.recharge.value) < 6 ? "+" : ""}
+    ]`;
```

Comment:
Can't test it at the moment because activities broke recharge, but wouldn't this lead to extra space between the brackets (`Recharge [ 5+ ]`)?

Reply:
My IDE was complaining but I think it is wrong and you are right.